int8 cls_id
float32 cls_confidence
float32 lpt
float32 rpt
float32 cdir 
float32 vel